package.json

```diff
@@ -4,26 +4,23 @@
   "description": "monorepo for ngrx development",
   "scripts": {
     "precommit": "lint-staged",
-    "bootstrap": "lerna bootstrap",
     "build": "bazel build ...",
     "deploy:builds": "ts-node ./build/deploy-build.ts",
     "test:unit": "node ./tests.js",
     "test": "nyc yarn run test:unit",
-    "clean": "git clean -xdf && yarn && yarn run bootstrap",
+    "clean": "git clean -xdf",
     "cli": "ng",
     "coverage:html": "nyc report --reporter=html",
     "copy:dist": "ncp dist/ ./node_modules/@ngrx/",
     "example:start": "yarn run cli serve",
     "example:start:aot": "yarn run cli serve --prod",
-    "example:test": "jest --watch",
-    "example:build:prod": "yarn cli build --prod --base-href \"/platform/example-app/\" --output-path \"./example-dist/example-app\"",
+    "example:test": "jest -c projects/example-app/jest.config.js --watch",
+    "example:build:prod": "yarn cli build --prod --base-href \"/platform/example-app/\" --output-path \"./projects/example-app/dist\"",
```

Comment:
I might be wrong but do we need the `output-path` and is this one correct?
In the `angular.json`, the following is specified `"outputPath": "dist/example-app",`

Reply:
Fixed